src/main/python/api.py

```diff
@@ -0,0 +1,44 @@
+from flask import Flask, jsonify, request
+import argparse
+# FIXME: separate this out to a classifier class where we can switch out the models
+from pyserini import Pyserini
+from jaccard import Jaccard
+
+app = Flask(__name__)
+
+@app.route("/", methods=['GET'])
+def hello():
+    return "Hello! The server is working properly... :)"
+
+@app.route('/answer', methods=['POST'])
+def answer():
+    try:
+        req = request.get_json(force=True)
+        question = req["question"]
+        print("Question: %s" % (question))
```

Comment:
Python doesn't recommend using the `%` operator anymore. Should use string.format instead, see https://docs.python.org/3.0/whatsnew/2.6.html#pep-3101-advanced-string-formatting.

```python
print("Question: {}".format(question))
```